trade-finance-manager-ui/templates/case/parties/parties.njk

```diff
@@ -10,6 +10,8 @@
 {% extends "case/case.njk" %}
 {%- from "govuk/components/tag/macro.njk" import govukTag -%}
 
+{% block pageTitle -%}Deal {{ deal.details.ukefDealId }} - Parties{%- endblock %}
```

Comment:
Multiline formatting please.

Reply:
Changed all pageTitle blocks to multiline format.